backends/apple/coreml/runtime/delegate/backend_delegate.h

```diff
@@ -28,7 +28,7 @@ class BackendDelegate {
         // Max models cache size in bytes.
         size_t max_models_cache_size = 10 * size_t(1024) * size_t(1024) * size_t(1024);
         // If set to `true`, delegate pre-warms the most recently used asset.
-        bool should_prewarm_asset = true;
+        bool should_prewarm_asset = false;
```

Comment:
Why do we need this change @cymbalrush?

Reply:
This feature was pre-warming the most recently used assets, which could impact the delegate initialization time and the first model load time. This optimization was beneficial only for the CPU backend and only when the subsequent asset load remained unchanged. So, disabling it by default.

Reply:
Thanks @cymbalrush, it's worth documenting the above behavior (on Line 30?)

Reply:
Let me create another PR for this.